tests/test_stac_ingestion.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import pystac
+from pystac import STACValidationError
+
+# Test validating the collection
+def test_validate_collection(test_collection):
+    try:
+        pystac.validation.validate_dict(test_collection)
+    except STACValidationError:
+        pytest.fail("Validation failed for test_collection")
+
+
+# Test validating the item
+def test_validate_item(test_item):
+    try:
+        pystac.validation.validate_dict(test_item)
+    except STACValidationError:
+        pytest.fail("Validation failed for test_item")
+
+
+# Test inserting collection
+def test_insert_collection(
+    stac_ingestion_instance, authentication_token, test_collection
+):
+    response = stac_ingestion_instance.insert_collection(
+        authentication_token, test_collection
+    )
+    assert response.status_code in [200, 201], "Failed to insert the test_collection"
```

Comment:
same here, let's provide which ever error information we get from the response, instead of just "Failed to insert the test_collection".

Reply:
Is this better:

`assert response.status_code in [200, 201], f"Failed to insert the test_collection {response.text}"`